packages/store/package.json

```diff
@@ -36,17 +36,13 @@
   ],
   "peerDependencies": {
     "@ember-data/graph": "workspace:5.3.0-alpha.4",
-    "@ember-data/json-api": "workspace:5.3.0-alpha.4",
     "@ember-data/legacy-compat": "workspace:5.3.0-alpha.4",
     "@ember-data/model": "workspace:5.3.0-alpha.4",
```

Comment:
this PR is complete when we can also remove this.

legacy-compat and graph will be handled in a separate PR